CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
 # Contribution
 
 ## Table of Contents
+* [Permission levels](#permissions)
 * [Labels](#labels)
 * [Issues & Labels](#issues)
 * [Pull Requests](#prs)
 * [Pull Requests Approval Process](#approvals)
 * [Idle Pull Requests Policy](#idle)
 * [Unresolved Pull Requests Policy](#unresolved)
 
+<a name="permissions"></a>
+### Permission levels
+There are different permissions (levels) available for contributing into CNTT CNTT:
+- **Outside Collaborators**: This includes general public. 
+  - Outside collaborators can add comments to issues and Pull Requests.
+- **Read, Triage, Write, and Maintain** Permissions, which are explained in [here](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization), will be assigned as follows:
+  - **Read** will be assigned to those who are interested to create issues, join the discussion and provide feedback.
+  - **Triage** will be assigned to active contributors who create issues, pull requests and review them.
+  - **Writes** will be assigned to Workstream Co-Leads to allow them to mange issues and pull reuquest, approve them and be auto requested for review.
```

Comment:
typo: mange -> manage
reuquest -> request

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1401/commits/b9f9cd73c66f3d9a314170f4e03c50bd2db19999